packages/forms-web-app/__tests__/unit/routes/index.test.js

```diff
@@ -56,7 +57,8 @@ describe('routes/index', () => {
     expect(use).toHaveBeenCalledWith('/document', documentRouter);
 
     expect(use).toHaveBeenCalledWith('/before-you-start', beforeYouStartRouter);
+    expect(use).toHaveBeenCalledWith('/submit-appeal', submitAppealRouter);
 
-    expect(use.mock.calls.length).toBe(15);
+    expect(use.mock.calls.length).toBe(16);
```

Comment:
Let's remove those number checks cause they are helpless. We are already checking the routes so no reason to count them